tools/ci_check_duplicate_usb_vid_pid.py

```diff
@@ -63,6 +63,7 @@
         "espressif_esp32s2_devkitc_1_n8r2",
     ],
     "0x239A:0x102E": ["weact_studio_pico", "weact_studio_pico_16mb"],
+    "0x00111011:0x00C60001": ["weact_esp32c6_n4", "weact_esp32c6_n8"],
```

Comment:
Let's just give two different creation ids for these. We have plenty of numbers. Maybe the last digit could be flash size?

Reply:
Alright, changed, as specified in the Creators ID pr. This entry in the script was removed.